src/style.css

```diff
@@ -2,2 +1,69 @@
+body,
+html {
+  padding: 0;
```

Comment:
Using the '*' selector is generally not recommended as it can impact performance. This is because it applies the styles to all elements on the page. Instead, you should target specific elements that require the styles. This is a requirement from the checklist: 'Don't use * selector (it impacts performance)'